components/ai_chat/resources/page/components/filter_menu/attachments_menu.tsx

```diff
@@ -41,6 +41,7 @@ const icon: Record<ReturnType<typeof getType>, string> = {
 export default function TabsMenu() {
   const aiChat = useAIChat()
   const conversation = useConversation()
+  const [selected, setSelected] = React.useState<Attachment | null>(null)
```

Comment:
formatting (extra indentation), wonder why this isn't getting caught

Reply:
removed